tests/phpunit/tests/block-template-utils.php

```diff
@@ -102,8 +107,14 @@ public function test_build_block_template_result_from_post() {
 		$this->assertSame( 'Description of my template', $template->description );
 		$this->assertSame( 'wp_template', $template->type );
 		$this->assertSame( self::$template_post->post_modified, $template->modified, 'Template result properties match' );
+	}
 
-		// Test template parts.
+	/**
+	 * @ticket 54335
+	 *
+	 * @covers ::_build_block_template_result_from_post
+	 */
+	public function test_build_block_template_result_from_post_template_part() {
```

Comment:
See https://github.com/WordPress/wordpress-develop/pull/5396/files#r1345811456.

🔢 Similar instances below (`test_build_block_template_result_from_file_template` etc)

Reply:
It should be better after f0250e35660ab87bd2c78c497908365a8d9d278d.